lib/components/DevCards.js

```diff
@@ -30,22 +32,78 @@ var DevCards = React.createClass({
   propTypes: {
     catalog: React.PropTypes.object.isRequired
   },
+  getInitialState: function() {
+    return {
+      namespace: null,
+      card: null
+    };
+  },
+  componentDidMount: function() {
+    var setState = this.setState.bind(this);
+
+    // Listening to URL changes
+    this.release = history.listen(function(location) {
+      var match = router.match(location.pathname);
```

Comment:
Might as well move it into `routes.js` actually, leaving the routes list unexported.